tools/testsys/src/aws_resources.rs

```diff
@@ -465,10 +465,7 @@ where
         .send()
         .await?
         .images;
-    let images: Vec<&Image> = describe_images
-        .iter()
-        .flat_map(|image| identity(image))
-        .collect();
+    let images: Vec<&Image> = describe_images.iter().flat_map(identity).collect();
```

Comment:
From clippy in actions workflow:
```
error: use of `flat_map` with an identity function
   --> testsys/src/aws_resources.rs:468:54
    |
468 |     let images: Vec<&Image> = describe_images.iter().flat_map(identity).collect();
    |                                                      ^^^^^^^^^^^^^^^^^^ help: try: `flatten()`
    |
    = note: `-D clippy::flat-map-identity` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#flat_map_identity
```

Reply:
I had to do some more reading and testing, but think it's good now. Still a little new on rust, so if someone can confirm everything looks good now, I'll hold off on merging to give folks a chance to take a look.